### DISCUSSION TITLE
文件参数理解

### DISCUSSION THREAD

**Discussion by Hold-on-li** [PaddlePaddle/PaddleOCR#12176] [Q&A]
![image](https://user-images.githubusercontent.com/87969070/135057338-1e35ce64-49ca-401c-8f76-307dfa469379.png)
请问这两个参数是不是对应着识别推理的识别个数和最大的文本长度？在什么场景下设置，麻烦告知，谢谢啦！

**Comment by Gmgge** [comment]
在此处的最下方进行了参数的解释https://github.com/PaddlePaddle/PaddleOCR/blob/4a1f2e3649a75b5a3a2edc7b04f6947ac5c28714/doc/doc_ch/whl.md
什么场景下设置？推理的场景呀。

**Comment by Hold-on-li** [comment]
> 在此处的最下方进行了参数的解释https://github.com/PaddlePaddle/PaddleOCR/blob/4a1f2e3649a75b5a3a2edc7b04f6947ac5c28714/doc/doc_ch/whl.md 什么场景下设置？推理的场景呀。
是的 在推理的场景下进行

**Comment by LDOUBLEV** [comment]
rec_batch_num: 识别模型预测的batch size
max_text_length: 基于attention预测的最大文本长度

**Comment by Hold-on-li** [comment]
> rec_batch_num: 识别模型预测的batch size max_text_length: 基于attention预测的最大文本长度

好的，感谢解答。请问有没有参数可以限制识别单张图像中的识别结果个数的吗？类似于检测过程中的max_candidates
